### PR TITLE
Override Content-type Header configuration added

### PR DESCRIPTION
**Pull request by r00z**
Fix for the issue: https://github.com/gravitee-io/issues/issues/1024

### CODE REVIEW

File: src/main/java/io/gravitee/policy/json2json/configuration/JsonToJsonTransformationPolicyConfiguration.java [hunk 1]
```diff
@@ -27,6 +27,9 @@
 
     private String specification;
 
+    //by default, we override the content-type to be backward compatible
+    private Boolean overrideContentType = true;
```
Comment (brasseld) [inline src/main/java/io/gravitee/policy/json2json/configuration/JsonToJsonTransformationPolicyConfiguration.java]:
```suggestion
    private boolean overrideContentType = true;
```

Comment (r00z) [reply src/main/java/io/gravitee/policy/json2json/configuration/JsonToJsonTransformationPolicyConfiguration.java]: updated

**Comment by brasseld** [comment]
Please squash the two commits into a single one and reference the issue in the commit message.

**Comment by r00z** [comment]
updated.

**Pull request merged**